.azure/pipelines/jobs/default-build.yml

```diff
@@ -91,7 +92,15 @@ jobs:
       ${{ if eq(parameters.agentOs, 'macOS') }}:
         vmImage: macOS-10.14
       ${{ if eq(parameters.agentOs, 'Linux') }}:
-        vmImage: ubuntu-16.04
+        ${{ if eq(parameters.useHostedUbuntu, true) }}:
+          vmImage: ubuntu-16.04
+        ${{ if eq(parameters.useHostedUbuntu, false) }}:
+          ${{ if eq(variables['System.TeamProject'], 'public') }}:
+            name: NetCorePublic-Pool
+            queue: buildpool.ubuntu.1604.amd64.open
+          ${{ if eq(variables['System.TeamProject'], 'internal') }}:
+            name: NetCoreInternal-Pool
+            queue: buildpool.ubuntu.1604.amd64
```

Comment:
@ilyas1974 and @riarenas how are we messing up here? These jobs are trying to run on Windows agents.

Reply:
Is `useHostedUbuntu` somehow undefined here so it is using the default?

Reply:
My suspicion based on https://helix.dot.net/api/2019-06-17/info/queues (now that I look closely) is queue names are case-sensitive. I just pushed a commit to see if that's the case.